app/src/main/java/de/danoeh/antennapod/fragment/HomePageFragment.java

```diff
@@ -30,11 +38,48 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         super.onCreateView(inflater, container, savedInstanceState);
-        View root = inflater.inflate(R.layout.home_page, container, false);
-
+        if (getActivity() instanceof MainActivity) {
+            ((MainActivity) getActivity()).getSupportActionBar().setTitle("Home");
+        }
+        try {
```

Comment:
Do you really need a try-catch here? Explain to me if you do.

Reply:
Yes, that was the solution to fix the back button functionality. Without it, I kept getting an error saying that the class couldn't be inflated.